appsec/src/extension/tags.c

```diff
@@ -485,12 +504,17 @@ static bool _add_all_ancillary_tags(
 
 // NOLINTNEXTLINE(bugprone-easily-swappable-parameters)
 static void _add_basic_tags_to_meta(
-    zval *nonnull meta, const zend_array *nonnull _server)
+    zval *nonnull meta, const zend_array *nonnull _server, bool all_headers)
```

Comment:
Another option, instead of a boolean, could be to pass a pointer to the header hash table.

Reply:
I thought about that but I chose this approach